openapi.yaml

```diff
@@ -3667,6 +3710,9 @@ paths:
         * If there is more than one Config profile, the last booted config will be used.
         * If there is more than one Config profile and none were the last to be booted (because the
           Linode was never booted or the last booted config was deleted) an error will be returned.
+
+        **Bare Metal Beta**
+        Boots a Linode identified by the ID of the Bare Metal Linode you have permission to boot.
```

Comment:
This is also the same as normal Linodes; the only difference for boot/reboot is that Config IDs are not accepted (are Bare Metal Instances don't have configs).